src/main/java/de/tum/in/www1/artemis/web/rest/admin/iris/AdminIrisSettingsResource.java

```diff
@@ -14,7 +14,7 @@
 /**
  * REST controller for managing {@link IrisSettings}.
  */
-@Profile("iris")
+@Profile("admin & iris")
```

Comment:
Do we have an admin profile now?

Reply:
No, we don't. These are remnants of changes that were reverted. I will remove these occurences from the code